vital/types/detected_object.h

```diff
@@ -118,7 +140,28 @@ class VITAL_EXPORT detected_object
    *
    * @param bbox Bounding box for this detection.
    */
-  void set_bounding_box( const bounding_box_d& bbox );
+  void set_bounding_box( bounding_box_d const& bbox );
+
+
+  /**
+   * @brief Get geo_point from this detection.
+   *
+   * The geo_point for this detection is returned. This point is in
+   * world coordinates. A default constructed (invalid) geo_point
```

Comment:
Saying that a `geo_point` is in world coordinates is sort of redundant, but eh, if it floats your boat... :slightly_smiling_face: